app/views/documents/history.html.erb

```diff
@@ -8,21 +8,25 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <% @timeline_entries.each do |entry| %>
-      <div class="app-timeline-entry govuk-!-margin-bottom-6">
-        <% if displayed_edition_number != entry.edition.number %>
-          <h3 class="govuk-heading-m"><%= t "documents.history.edition_header", number: entry.edition.number.ordinalize %></h3>
-          <% displayed_edition_number = entry.edition.number %>
+    <% @timeline_entries.each_with_index do |entry, index| %>
+
+      <% if displayed_edition_number != entry.edition.number %>
+        <% if index > 0 %>
+          <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--xl">
         <% end %>
+        <h3 class="govuk-heading-m"><%= t "documents.history.edition_header", number: entry.edition.number.ordinalize %></h3>
+        <% displayed_edition_number = entry.edition.number %>
+      <% end %>
 
+      <div class="app-timeline-entry <%= "app-timeline-entry--highlighted" if entry.details %>">
```

Comment:
I feel like it's a bit presumptuous for us to rely on the presence of a details object as the indication that we want to show a highlight. For instance that convention is just about to be broken in https://github.com/alphagov/content-publisher/pull/1644

I think it would be safer to have an array of highlighted_entry_types and then we can ask `<%= "app-timeline-entry--highlighted" if highlighted_entry_types.include?(entry.entry_type) %>">`

Reply:
I might have misunderstood the card description here:

> Notes or history events where a user entered additional content, for example: Internal notes, Withdrawn, Backdated, removed. (this is grey by a different background as shown in the prototype)

I read 'entered additional content' as regardless of the entry type we only highlight if there is content. Happy to swap to an type-based approach if that's what we want.

Reply:
Yeah I think we should as we can't guarantee a details object being present means that we'll show the user extra information.

Reply:
I doubled check this with Simon and the design intention is to highlight the entries with content, not based on type. If I understand well checking for `entry.details` could be flaky, is there another way to check if a timeline entry has user-generated comments on it?

Reply:
Since #1644 got merged I rebased this to master and updates the highlighting logic as discussed on the offline review. It also covers the view test discussed in the comment below.